modules/ingestor/src/service/format.rs

```diff
@@ -49,41 +49,47 @@ impl<'g> Format {
     where
         S: Stream<Item = Result<Bytes, anyhow::Error>> + Send + 'static,
     {
+        let mut buffer = Vec::new();
+        let mut s = pin!(StreamReader::new(
+            stream.map_err(|e| io::Error::new(io::ErrorKind::Other, format!("{e:?}"))),
+        ));
+        s.read_to_end(&mut buffer).await?;
```

Comment:
@bobmcwhirter i'm not sure about this, but it's a whole lot simpler than the bridge stuff deleted below.

Reply:
Seems good to me. In my ish, I've been using the sync std::io `read_to_end()` which is probably sub-optimal within the context of Tokio.  But on the importer side, it seems we don't async all that much.